README.md

```diff
@@ -16,6 +16,7 @@
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
_I suggest you also delete this header text here also since there is no corresponding section. Please fix this 🙏_